frontend/admin/.env.example

```diff
@@ -25,11 +25,7 @@ QUEUE_CONNECTION=sync
 
 API_URI="/graphql"
 
-# TODO: some sort of logout landing page
```

Comment:
Oops, could you please revert this file?  That was just a workaround for your environment for now.